ddtrace/contrib/_trace_utils_llm.py

```diff
@@ -37,8 +37,16 @@ def __init__(self, config, stats_url, site, api_key):
             interval=float(os.getenv("_DD_%s_LOG_WRITER_INTERVAL" % self._integration_name.upper(), "1.0")),
             timeout=float(os.getenv("_DD_%s_LOG_WRITER_TIMEOUT" % self._integration_name.upper(), "2.0")),
         )
+        self._llmobs_writer = LLMObsWriter(
+            site=site,
+            api_key=api_key,
+            app_key=app_key,
+            interval=float(os.getenv("_DD_%s_LLM_WRITER_INTERVAL" % self._integration_name.upper(), "1.0")),
+            timeout=float(os.getenv("_DD_%s_LLM_WRITER_TIMEOUT" % self._integration_name.upper(), "2.0")),
```

Comment:
Do we need to make the interval and timeout configurable via internal configurations? When will we need to toggle these values?

Reply:
These config values are only relevant to specific user cases for sending llmobs payloads at different intervals/timeouts. Most of the time these won't need to be toggled.

What do you mean by internal configurations?